modules/oe_content_event/config/install/field.field.node.oe_event.oe_subject.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_subject
+    - node.type.oe_event
+  module:
+    - rdf_skos
+id: node.oe_event.oe_subject
+field_name: oe_subject
+entity_type: node
+bundle: oe_event
+label: Subject
+description: ''
+required: true
+translatable: true
```

Comment:
This field is not translatable by the user.

Reply:
Fixed. Thanks!